src/stories/Toggle/Toggle.stories.tsx

```diff
@@ -22,18 +22,20 @@ export default {
     }
 } as Meta;
 
-const toggleOptions: ToggleOption[] = [
+const toggleOptions: ToggleOption<string>[] = [
```

Comment:
Do you think it'd be a good idea to also have an example in stories showing a non-string usage now?

Reply:
I could make it a `number` or something to show a difference, but since it's just the value for the state there's nothing visual to show in this or a separate story. The typing was more just to free up the parent component's state to store what it wanted, like an enum value or something, but also tie it to the same type so onChange would know what it should be better than if I chose `any` instead. If you think it's helpful to change from string in this story I will, though.